### PR TITLE
Fixes for 1.0

### PR DESCRIPTION
**Pull request by miguelraz**
:bangbang: :gun:  pew pew!

Also cleaned up the tests into neater testsets.

### CODE REVIEW

**Comment by dpsanders** [comment]
You need to update REQUIRE and the Travis scripts I guess?

File: .travis.yml [hunk 1]
```diff
@@ -4,9 +4,10 @@ os:
...
 julia:
-  - 0.6
+  - 0.7
+  - 1.0
   - nightly
 notifications:
   email: false
 after_success:
-  - julia -e 'cd(Pkg.dir("FastRounding")); Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder())';
+  - julia -e 'Pkg.dir("FastRounding"); Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder())';
```
Comment (dpsanders) [inline .travis.yml]: `Pkg.dir` is deprecated.

File: test/runtests.jl [hunk 2]
```diff
@@ -13,10 +13,6 @@ function testrounding(op::Function, a::T, b::T, mode::RoundingMode) where T<: Sy
 
     c = setrounding(BigFloat, mode) do
             S(op(big(a), big(b)))
-        end
-    return c
-end
-=#
```
Comment (miguelraz) [inline test/runtests.jl]: Was this meant to be removed?

**Comment by JeffreySarnoff** [comment]
no .. putting it back

**Comment by miguelraz** [comment]
Got it.
I also refactored the `testsets` - is that a change you would prefer to keep?
We have merging conflicts.

**Pull request merged**